spec/custom_rules/profile.log

```diff
@@ -0,0 +1,2072 @@
+FUNCTION  <SNR>101_get_syn()
```

Comment:
is this committed by mistake?

Reply:
Oh lord, yeah it was. Good catch, let me remove it!